File: src/netcore-api/SIL.XForge.WebApi.Server/Models/Lexicon/LexTask.cs
namespace SIL.XForge.WebApi.Server.Models.Lexicon
{
    public class LexTask
    {
        public const string View = "view";
        public const string Dashboard = "dashboard";
        public const string GatherTexts = "gatherTexts";
        public const string Semdom = "semdom";
        public const string Wordlist = "wordlist";
        public const string Dbe = "dbe";
        public const string AddMeanings = "addMeanings";
        public const string AddGrammar = "addGrammar";
        public const string AddExamples = "addExamples";
        public const string Review = "review";
        public const string ImportExport = "importExport";
        public const string Configuration = "configuration";

        public bool Visible { get; set; } = true;
        public string Type { get; set; } = "";
    }
}

File: src/Api/Model/Languageforge/Lexicon/Config/LexTask.php
<?php

namespace Api\Model\Languageforge\Lexicon\Config;

class LexTask
{
    // task types
    const VIEW = "view";
    const DASHBOARD = "dashboard";
    const GATHERTEXTS = "gatherTexts";
    const SEMDOM = "semdom";
    const WORDLIST = "wordlist";
    const DBE = "dbe";
    const ADDMEANINGS = "addMeanings";
    const ADDGRAMMAR = "addGrammar";
    const ADDEXAMPLES = "addExamples";
    const REVIEW = "review";
    const IMPORTEXPORT = "importExport";
    const CONFIGURATION = "configuration";

    public function __construct()
    {
        $this->visible = true;
        $this->type = "";
    }

    /** @var boolean */
    public $visible;

    /** @var string */
    public $type;
}
